gum/backend-qnx/gummemory-qnx.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "gummemory.h"
+
+#include "gumqnx-priv.h"
+
+#include <fcntl.h>
+#include <gio/gio.h>
+#include <stdio.h>
+#include <sys/mman.h>
+#include <sys/procfs.h>
+
+static gboolean
```

Comment:
Can we move this function down just before _gum_page_protection_from_posix? It is lower level than the public functions following it.